recipes/msgpack/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class MsgpackConan(ConanFile):
+    name = "msgpack"
```

Comment:
Shouldn't the name be `msgpack-c`? But `msgpack` is alright too.

Reply:
As said above, I stole the recipe from bincrafters, I kept most things as they were

Reply:
The name is fine! I was just mentioning it.